app/src/main/java/tech/ula/utils/FilesystemUtility.kt

```diff
@@ -96,9 +96,11 @@ class FilesystemUtility(
 
     fun moveAppScriptToRequiredLocations(appName: String, appFilesystem: Filesystem) {
         // TODO add error cases
+        val fileNameToForceAppScriptToExecuteLast = "zzzzzzzzzzzzzzzz.sh"
         val appScriptSource = File("$applicationFilesDirPath/apps/$appName/$appName.sh")
         val appScriptSupportTarget = File("$applicationFilesDirPath/${appFilesystem.id}/support/$appName.sh")
-        val appScriptProfileDTarget = File("$applicationFilesDirPath/${appFilesystem.id}/etc/profile.d/$appName.sh")
+        val appScriptProfileDTarget = File("$applicationFilesDirPath/${appFilesystem.id}" +
```

Comment:
What does D stand for?

Reply:
In `appScriptProfileDTarget`

Reply:
The directory where these scripts go is called `profile.d` as in `/etc/profile.d`

Reply:
Makes sense 👍